app/views/content/life-as-a-teacher/change-careers/_categories.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <section class="category col-space-m">
+    <section class="container category__cards main-section">
+      <nav class="category__nav-cards">
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Support with changing careers to teaching",
```

Comment:
This order feels slightly odd to me - would it be better as:

* Why change (benefits)
* How to change
* Support to change